core/repository/sparql/src/main/java/org/eclipse/rdf4j/repository/sparql/federation/SilentIteration.java

```diff
@@ -34,9 +39,20 @@ protected BindingSet getNextElement() throws QueryEvaluationException {
 				return iter.next();
```

Comment:
I know it's not required by our styleguide, but I do prefer using { } for if/else/while.

Reply:
Agreed. Actually it is a convention for the RDF4J, it's just not explicitly enforced by the formatter because in Eclipse, this option is not part of the formatting rules (so I can't make it part of the maven formatter plugin, which uses Eclipse formatter under the hood).

Btw if you're using Eclipse, you can configure it to automatically add braces on save, by going to 'Preferences' -> 'Editor' -> 'Save Actions'. Enable 'Additional actions', click 'Configure...' and in the 'Code Style' dialog enable the option 'Use blocks in if/while/for/do statements'.  

Or if you can prefer you can use an Eclipse clean up profile to do the same thing.